### ISSUE TITLE
A text/html be recognized as an image

### ISSUE THREAD

**Issue by takayama-lily** [nodeca/probe-image-size#60]
```js
var probe = require('probe-image-size')
probe("https://nodejs.org/dist/latest-v14.x/docs/api/stream.html").then(console.log)
```

output:

```
{
  width: 24,
  height: 24,
  type: 'svg',
  mime: 'image/svg+xml',
  wUnits: 'px',
  hUnits: 'px',
  url: 'https://nodejs.org/dist/latest-v14.x/docs/api/stream.html'
}
```

**Comment by puzrin** [comment]
SVG inside HTML. Nice catch :).

I think, extra check for "no html/head/body patterns before <svg...>" should solve problem.

**Comment by rlidwka** [comment]
Fixed, thanks for reporting.

**Comment by puzrin** [comment]
Released 7.2.0
